components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -193,6 +193,41 @@ public Response createAPI(APIDTO body, String oasVersion, MessageContext message
             throws APIManagementException{
         URI createdApiUri;
         APIDTO createdApiDTO;
+
+        boolean isOrgValidationEnabled = false;
+
+        Map<String, String> apiCreatorValidationConfigs = APIManagerConfiguration.getApiCreatorValidationProperties();
+        if (apiCreatorValidationConfigs != null &&
+                apiCreatorValidationConfigs.containsKey(APIConstants.ENABLE_API_CREATOR_ORG_VALIDATION)) {
+            isOrgValidationEnabled = Boolean.valueOf(apiCreatorValidationConfigs.get(
+                    APIConstants.ENABLE_API_CREATOR_ORG_VALIDATION));
+            log.debug("Organization validation for API creator is enabled");
+        }
```

Comment:
Can we create a util method for this and reuse in other places?
isBackendUrlOrgValidationEnabled()